server/tests/1userSignup.test.js

```diff
@@ -16,6 +16,14 @@ const user = {
   password: 'Qwertyui0p',
   confirmPassword: 'Qwertyui0p',
 };
+const user2 = {
```

Comment:
Hi! clinton, well done. I like the way you structured your test. But I think you forgot to change that user2 in the test file to a more descriptive name as Mark requested.

Reply:
This is not my test file but I will make the necessary changes.